doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -255,18 +287,57 @@ We are using a User OS based on `Clear Linux`_.
 
   .. code-block:: none
 
-     -s 3,virtio-blk,/root/clear-22780-kvm.img
-     -k /lib/modules/kernel/org.clearlinux.pk414-standard.4.14.47-44
+     -s 3,virtio-blk,/root/clear-24040-kvm.img
+     -k /lib/modules/kernel/org.clearlinux.pk414-standard.4.14.57-69
 
 Start the User OS (UOS)
 ***********************
 
 You are now all set to start the User OS (UOS)
 
-.. code-block:: none
+ .. code-block:: none
 
    sudo /usr/share/acrn/samples/nuc/launch_uos.sh
 
 **Congratulations**, you are now watching the User OS booting up!
 
+
+Enabling network sharing 
+************************
+
+After booting up SOS and UOS, enabling network sharing is must to test networking in UOS side. 
+enabling the tap and networking bridge in the SOS, you can create a script file to setup.
+below is an example for you reference:(verifed in Ubuntu 14.04 and 16.04 as the SOS) 
+ 
+ .. code-block:: none
+  
+    #!/bin/bash
+    #setup bridge for uos network
+    br=$(brctl show | grep acrn-br0)
+    br=${br-:0:6}
+    ip tuntap add dev acrn_tap0 mode tap
+    
+    taps=$(ifconfig | grep acrn_ | awk '{print $1}')
+  
+    # if bridge not existed
+    if [ "$br"x != "acrn-br0"x ]; then
+    #setup bridge for uos network
+    brctl addbr acrn-br0
+    brctl addif acrn-br0 enp3s0
+    ifconfig enp3s0 0
+    dhclient acrn-br0
+    # add existing tap devices under the bridge
+      for tap in $taps; do
+        ip tuntap add dev acrn_$tap mode tap
+        brctl addif acrn-br0 $tap
+        ip link set dev $tap down
+        ip link set dev $tap up
+      done
+    fi
+  
+    brctl addif acrn-br0 acrn_tap0
+    ip link set dev acrn_tap0 up
+ 
+Enabling USB keyboard and mouse for UOS, please :ref:`getting-started-apl-nuc` 
```

Comment:
This comment seems out of place.  We're in the Enabling Network Sharing section and you're talking about the USB keyboard and mouse.  Maybe add a new section heading and rewrite this as:

    Enabling USB keyboard and mouse
    *******************************

    Please refer to :ref:`getting-started-apl-nuc` for enabling the
    USB keyboard and mouse for the UOS.